mlflow/cli.py

```diff
@@ -237,8 +237,14 @@ def _validate_static_prefix(ctx, param, value):  # pylint: disable=unused-argume
               help="Additional command line options forwarded to gunicorn processes.")
 @click.option("--waitress-opts", default=None,
               help="Additional command line options for waitress-serve.")
-def server(backend_store_uri, default_artifact_root, host, port,
-           workers, static_prefix, gunicorn_opts, waitress_opts):
+@click.option("--activate-scheduler", default=False,
```

Comment:
This 3 options are very specific to your DB task.
1/ It will not scale (if we add new periodic task)
2/ And in this case the naming is not correct. It should refer to (clean_metrics_db)

I think we reach the limit of click.option. What about to have a list of periodic job to activate? and a config file